docs/buildah-manifest-add.md

```diff
@@ -0,0 +1,86 @@
+# buildah-manifest-add "1" "September 2019" "buildah"
+
+## NAME
+
+buildah\-manifest\-add - Add an image to a manifest list or image index.
+
+## SYNOPSIS
+
+**buildah manifest add** *listName* *imageName*
+
+## DESCRIPTION
+
+Adds the specified image to the specified manifest list or image index.
+
+## RETURN VALUE
+
+The list image's ID and the digest of the image's manifest.
+
+## OPTIONS
+
+**--all**
+
+If the image which should be added to the list or index is itself a list or
+indexes, add all of its contents to the local list.  By default, only one image
```

Comment:
on the fence myself, but a nit for consideration.   "its contents" to "the contents".

Reply:
Changing "its" to "the".  Correcting "indexes" to "index".